dynamite/plotter.py

```diff
@@ -2076,7 +2076,8 @@ def orbit_distribution(self,
                            equal_weighted_orbits=False,
                            orientation='horizontal',
                            figtype='.png',
-                           subset='all'):
+                           subset='all',
+                           getdata='False'):
```

Comment:
change string to boolean, i.e. `'False'` --> `False`

Reply:
Thank you! Fixed it